tests/linalg_test.py

```diff
@@ -1318,7 +1318,9 @@ def testExpmGrad(self, n, dtype, rng_factory):
      rng = rng_factory(self.rng())
      jtu.skip_if_unsupported_type(dtype)
      a = rng((n, n), dtype)
-     jtu.check_grads(jsp.linalg.expm, (a,), modes=["fwd"], order=1)
+     def expm(x):
+       return jsp.linalg.expm(x, upper_triangular=False, max_n_squarings=16)
+     jtu.check_grads(expm, (a,), modes=["fwd", "rev"], order=1)
```

Comment:
could we test `order=2`, also? I think the gradient should also be differentiable.

Reply:
Done.